scan.go

```diff
@@ -272,9 +265,53 @@ func scan(pkgPatterns []string, scanDir string) ([]*unit.SourceUnit, error) {
 			Ops:          map[string]*srclib.ToolRef{"depresolve": nil, "graph": nil},
 		})
 	}
-	if err := cmd.Wait(); err != nil {
+
+	return units, nil
+}
+
+func scanForPackages(dir string) ([]*build.Package, error) {
+	var pkgs []*build.Package
+
+	pkg, err := buildContext.ImportDir(dir, 0)
+	if _, isNoGoError := err.(*build.NoGoError); err != nil && !isNoGoError {
 		return nil, err
 	}
+	if err == nil {
+		pkgs = append(pkgs, pkg)
+	}
 
-	return units, nil
+	infos, err := ioutil.ReadDir(dir)
+	if err != nil {
+		return nil, err
+	}
+	for _, info := range infos {
+		name := info.Name()
+		fullPath := filepath.Join(dir, name)
+		if info.IsDir() && ((name[0] != '.' && name[0] != '_' && name != "testdata") || (strings.HasSuffix(fullPath, "Godeps/_workspace") && !config.SkipGodeps)) {
```

Comment:
Actually no. ;-) The packages have there normal import path relative to `_workspace`, because `_workspace` is added to the GOPATH just like Godeps would do. The code here is just scanning for absolute directories which potentially contain Go packages. `go/build` then figures out the import path relative to the closest GOPATH.